lib/conjur/fpm/Dockerfile

```diff
@@ -5,14 +5,9 @@ RUN apt-get update -y && \
     apt-get install -y software-properties-common && \
     apt-add-repository -y ppa:brightbox/ruby-ng && \
     apt-get update -y && \
-    apt-get install -y build-essential git libpq5 libpq-dev ruby2.3 ruby2.3-dev libffi-dev
+    apt-get install -y build-essential git libpq5 libpq-dev ruby2.6 ruby2.6-dev libffi-dev
 
-
-# Configure bundler and gem the way the ruby:2.3 Dockerfile does, as of
-# https://github.com/docker-library/ruby/blob/c88f3a67da720bfa9fb1717960d90fd5db11c757/2.3/Dockerfile#L41-L54
-ENV BUNDLER_VERSION 1.11.2
-
-RUN gem install --no-rdoc --no-ri bundler:$BUNDLER_VERSION ruby-xz:0.2.3 fpm
```

Comment:
`ruby-xz` not needed any more? Could you add a note to the commit message explaining why?